### PR TITLE
Update build scripts to allow local builds with Docker, fix publishing of latest

### PR DESCRIPTION
**Pull request by hansgschossmann**
## Purpose

* Add build scripts to enable local container build
* update container with tag latest in mcr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,9 @@ The option `sph` shows and dumps a pn.json file that matches the configuration.
 The build scripts are for Azure DevOps and the container build is done in ACR. To use your own ACR you need to:
 
 - Create a **service connection** called azureiiot to the subscription/resource group in which your ACR is located
-- Set a variable called **azureContainerRegistry** with the name of your Azure Container Registry
+- Set a variable called **BUILD_REGISTRY** with the name of your Azure Container Registry
+
+Using `tools/scripts/build.ps1` you can also build with Docker Desktop locally.
```
Comment (luiscantero) [inline README.md:122]: Pls add sample call to docker-build.ps1 incl. required parameters, e.g.: tools/scripts/docker-build.ps1 -Path ./src -ImageName iot-edge-opc-plc:<TAG>

File: tools/scripts/acr-build.ps1 [hunk 2]
```diff
@@ -27,11 +27,6 @@ Param(
...
 )
 
-# use registry from environment if not passed in
-if ([string]::IsNullOrEmpty($Registry) -and ![string]::IsNullOrEmpty($env:azureContainerRegistry)) {
-    $Registry = $env:azureContainerRegistry
-}
-
 # Check path argument and resolve to full existing path
 if ([string]::IsNullOrEmpty($Path)) {
     throw "No docker folder specified."
```
Comment (luiscantero) [inline tools/scripts/acr-build.ps1:32]: This is a bit misleading. Most users would expect a docker folder to be the folder with the Dockerfile. But this actually needs the container.json file. We should improve the error message. Consider adding sample call with required params to readme.

File: tools/scripts/docker-build.ps1 [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+#>
+
+Param(
+    [string] $Path = ".",
+    [string] $ImageName,
+    [switch] $Debug
+)
+
+if ([string]::IsNullOrEmpty($ImageName)) {
+    throw "Must specifiy an image name"
```
Comment (luiscantero) [inline tools/scripts/docker-build.ps1:23]: Typo specify

File: README.md (hunk 1 above)
Comment (hansgschossmann) [reply README.md]: fixed

File: tools/scripts/docker-build.ps1 (hunk 3 above)
Comment (hansgschossmann) [reply tools/scripts/docker-build.ps1]: fixed

File: tools/scripts/acr-build.ps1 (hunk 2 above)
Comment (hansgschossmann) [reply tools/scripts/acr-build.ps1:32]: not fixing this one.

**Comment by hansgschossmann** [comment]
Fix build to allow local builds, update MCR latest container image, and update doc.

**Pull request merged**